project.clj

```diff
@@ -1,11 +1,11 @@
-(defproject puppetlabs/http-client "0.9.1-SNAPSHOT"
+(defproject puppetlabs/http-client "0.10.0-SNAPSHOT"
   :description "HTTP client wrapper"
   :license {:name "Apache License, Version 2.0"
             :url "http://www.apache.org/licenses/LICENSE-2.0.html"}
 
   :min-lein-version "2.7.1"
 
-  :parent-project {:coords [puppetlabs/clj-parent "0.6.1"]
+  :parent-project {:coords [puppetlabs/clj-parent "1.7.12"]
```

Comment:
This is the version of clj-parent for Irving, I assume that Johnson also has this problem? Should we branch at 1.x off of master and then upgrade master to clj-parent 2.3 for Johnson?

Reply:
Sounds good.  We need this fix for irving for performance issues in quite a few components.